Assets/Scripts/UI/UpgradeUI.cs

```diff
@@ -30,8 +30,8 @@ void Awake()
             setButton(choiceButtons[i], i);
             choiceButtons[i].GetComponent<UpgradeChoice>().choiceNum = i;
         }
-
-        gameObject.SetActive(false);
+		GetComponent<Canvas>().enabled = true;
```

Comment:
What does this do?

Reply:
The upgrade and death screen were always rendered in the editor view so it was annoying to edit UI stuff, its just making it visible again. It can be removed.